### DISCUSSION TITLE
Docker - help selecting a specific commit build

### DISCUSSION THREAD

**Discussion by archedraft** [TandoorRecipes/recipes#2965] [Q&A] [answered]
I am correctly on the develop branch but would like to switch to master. I need to wait for master to catch up to develop or I get errors.

I am currently using the develop commit 565a732ff01ae3054d3cde6052866f6325b2301a in an unRAID docker. My server is set to update dockers each night and I want to keep this enabled. That said, I want to keep Tandoor on this current commit until the master catches up.

I have tried multiple formats but I am just guessing and it keeps failing when I try them. Does anyone know how to specific a specific commit so that it won't update on me?

Attempts that didn't work:
vabene1111/recipes:develop@565a732ff01ae3054d3cde6052866f6325b2301a
vabene1111/recipes:develop@565a732
vabene1111/recipes:develop-565a732

**Comment by smilerz** [comment, marked as answer]
Commit specific releases aren't maintained.  You'll need to exclude tandoor from automatic docker updates.

**Reply by archedraft** [reply]
Thanks. I didn't realize I could set which dockers get excluded from the auto update in unRAID. I was able to exclude Tandoor for the time being. I appreciate your help!

**Reply by smilerz** [reply]
One other thing you could try is updating to beta and see if that works. If it does that will get you a shorter path to migrating to main branch.
